.github/workflows/tag-publish.yml

```diff
@@ -169,7 +182,7 @@ jobs:
         uses: ncipollo/release-action@v1
         id: create_release
         with:
-          tag: ${{ steps.tag.outputs.tag }}
```

Comment:
there is another place in this job step where `steps.tag.outputs.tag` is referenced, do we not need to change that as well? 

https://github.com/DataBiosphere/terra-scientific-pipelines-service/blob/main/.github/workflows/tag-publish.yml#L156

Reply:
oh good call, there were actually several other places that needed to be updated. thank you!